진료실 생성(진료실 생성할 때 진료항목도 함께 생성할 수 있음), 업데이트, 삭제 기능 추가. - hospitalOffice…

leemagnon opened the pull request:
….route.ts, hospitalOffice.service.ts 추가.

ReservationLog 모델 삭제.
모든 병원 조회 시 각 병원의 진료실(officeName)과 진료항목(treatmentName)도 함께 조회됨.
예약 status PENDING, ACCEPTED, REFUSED, TIMEOUT으로 나눌 예정.

HyoVeemo commented on src/db.ts:
```diff
@@ -10,7 +10,7 @@ class Db {
 
   constructor() {
-    this.path = `${__dirname}/.env`;
+    this.path = `/Users/rat2/T4server/.env`; ///Users/rat2/T4server/.env
```
이 부분 이렇게 하면 내 컴퓨터에는 rat2라는 파일이 없기 때문에 동작이 안될거같아!
절대경로 말고 상대경로로 부탁해!

leemagnon committed:
진료실 생성(진료실 생성할 때 진료항목도 함께 생성할 수 있음), 업데이트, 삭제 기능 추가. - hospitalOffice.route.ts, hospitalOffice.service.ts 추가.
ReservationLog 모델 삭제.
모든 병원 조회 시 각 병원의 진료실(officeName)과 진료항목(treatmentName)도 함께 조회됨.
예약 status PENDING, ACCEPTED, REFUSED, TIMEOUT으로 나눌 예정.

leemagnon committed:
병원이 예약 요청을 무시/수락/거절할 때와 예약 시간이 지났을 때의 status 변화 적용.
